yojson.opam

```
@@ -1,22 +1,30 @@
-opam-version: "1.2"
+opam-version: "2.0"
 maintainer: "martin@mjambon.com"
 authors: ["Martin Jambon"]
-homepage: "https://mjambon.github.io/mjambon2016/yojson-doc/Yojson.html"
-bug-reports: "https://github.com/mjambon/yojson/issues"
-dev-repo: "https://github.com/mjambon/yojson.git"
-available: [ ocaml-version >= "4.02.3"]
+homepage: "https://github.com/ocaml-community/yojson"
+bug-reports: "https://github.com/ocaml-community/yojson/issues"
+dev-repo: "git+https://github.com/ocaml-community/yojson.git"
+doc: "https://ocaml-community.github.io/yojson/"
 build: [
-  ["jbuilder" "subst" "-n" name] {pinned}
-  ["jbuilder" "build" "-p" name "-j" jobs]
+  ["dune" "subst" "-n" name] {pinned}
```

no need for `-n` with `dune subst` so that can be removed
```suggestion
  ["dune" "subst"] {pinned}
```

---

Thanks for the suggestion, applied in a542b6a. 👍